syntax = "proto3";

package WMS_um;

service WMSUm {
  rpc RequestUserIdViaJWTToken (Request) returns (Reply) {}
}

message Request {
  string jwt_token = 1;
}

message Reply {
  int64 user_id = 1;
}